src/services/animal/api.ts

```diff
@@ -58,6 +64,17 @@ const api = {
       .get();
   },
 
+  getNotOwnedByUser(
```

Comment:
Provavel que essa API fique sem uso

Reply:
I'll keep the API so that you can replace when merging with Infinite Scroll PR